src/types.ts

```diff
@@ -338,6 +338,10 @@ export type Connector = {
   credentials: ConnectorCredential[]
   hasMFA: boolean
   oauthUrl?: string
+  health: {
+    status: 'ONLINE' | 'OFFLINE'
```

Comment:
we also have `'UNSTABLE'` as a possible type (https://github.com/pluggyai/pluggy-api/blob/f517a96acc1a06f7a65f77dc570598ca18b568c6/src/models/types/connectorStatusType.ts#L4), can we please add it here too?